website/public/locales/th/chat.json

```diff
@@ -0,0 +1,4 @@
+{
+  "back_to_chat_list": "กลับไปรายการแชท",
+  "your_chats": "แชทของคุณ"
+}
```

Comment:
this file is duplicated?

https://github.com/LAION-AI/Open-Assistant/blob/main/website/public/locales/th/chat.json

Reply:
This can likely be resolved by just merging main into this PR branch